src/vs/workbench/browser/contextkeys.ts

```diff
@@ -214,8 +219,13 @@ export class WorkbenchContextKeysHandler extends Disposable {
 
 		if (activeControl) {
 			this.activeEditorContext.set(activeControl.getId());
+
+			const resource = activeControl.input.getResource();
```

Comment:
@sbatten this fails for diff editors. Use `toResource` (`src/vs/workbench/common/editor.ts`) with the option `supportSideBySide` to support it.